platform/android/src/jni.cpp

```diff
@@ -484,6 +484,13 @@ void nativeMoveBy(JNIEnv *env, jni::jobject* obj, jlong nativeMapViewPtr, jdoubl
                           jlong duration) {
     assert(nativeMapViewPtr != 0);
     NativeMapView *nativeMapView = reinterpret_cast<NativeMapView *>(nativeMapViewPtr);
+
+    mbgl::AnimationOptions animationOptions;
+    if (duration > 0) {
+       animationOptions.duration.emplace(mbgl::Milliseconds(duration));
+       animationOptions.easing.emplace(mbgl::util::UnitBezier { 0, 0.3, 0.6, 1.0 });
+    }
+
     nativeMapView->getMap().moveBy({dx, dy}, mbgl::Milliseconds(duration));
```

Comment:
`animationOptions` isn't being used here. Looks like you'll need to change `Map::moveBy` before that's possible.